src/components/Content/Content.js

```diff
@@ -9,6 +11,11 @@ import RecipePage from '../RecipePage/RecipePage';
 
 
 const Content = () => {
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(fetchAllRecipes());
+    }, [dispatch]);
```

Comment:
To trzeba będzie przenieść w odpowiednie miejsce (na listę przepisów, a z kolei na pojedynczym przepisie pobierać z bazy tylko ten jeden przepis).